server/schedulers/balance_region.go

```diff
@@ -94,6 +96,9 @@ func newBalanceRegionScheduler(opController *schedule.OperatorController, conf *
 		filter.StoreStateFilter{ActionScope: scheduler.GetName(), MoveRegion: true},
 		filter.NewSpecialUseFilter(scheduler.GetName()),
 	}
+	scheduler.targetFilters = []filter.Filter{
+		filter.NewLabelConstaintFilter("", []placement.LabelConstraint{{Key: "store-do-not-balance-to", Op: placement.NotExists}}),
```

Comment:
This method is not recommended, you can add parameters to this schedule, refer to https://github.com/pingcap/pd/pull/1937.